com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1373,7 +1384,7 @@ private void ConfigureSimulator()
 #endif
 #endif
 
-#if UNITY_EDITOR || DEVELOPMENT_BUILD
+#if UNITY_EDITOR || DEVELOPMENT_BUILD || MULTIPLAYER_TOOLS_1_1_0
```

Comment:
Should this `||` only occur if `NetSim` is actually included in the build? It's possible to have a release build without NetSim included right?

Reply:
You are right. Which means we will need the boilerplate stuff, urgh.